### ISSUE TITLE
no-unresolved | Allow `@` for image size naming

### ISSUE THREAD

**Issue by nickofthyme** [benmosher/eslint-plugin-import#1021]
Allow for the use of iOS sizing when naming image files using `@` (i.e. `[IMAGE]@3x.png`) with react-native

### Example folder structure
```
project
│   test.js
└───images
│   │   sun@1x.png
│   │   sun@2x.png
│   │   sun@3x.png
```

### Example uses
```js
// test.js

// wrong usage and no linting error
const sun2 = require('./images/sun@3x.png');

// correct usage but throws linting error
const sun1 = require('./images/sun.png');
```

### Example error in linter
![image](https://user-images.githubusercontent.com/19007109/36129395-35e41666-1024-11e8-81e4-42a1b6a2f3cd.png)

**Comment by ljharb** [comment]
Sounds like you need a custom resolver to teach eslint that your nonexistent path should pretend to exist.
